src/wizards/topics.ts

```diff
@@ -66,6 +66,14 @@ export async function addTopicWizard(clientAccessor: ClientAccessor, clusterSett
         } else {
             explorer.refresh();
             window.showInformationMessage(`Topic '${topic}' in cluster '${clusterName}' created successfully`);
+            // Select the created topic is done with TreeView#reveal
+            // 1. Show the treeview of the explorer (otherwise reveal will not work)
+            explorer.show();
+            // 2. the call of reveal must be done in a timeout

```

Comment:
the reveal() call must occur in a timeout,

Reply:
fixed